Makefile

```diff
@@ -18,6 +18,10 @@ add-suppliers:
 check-python-code:
 	docker compose -f tests/docker-compose.yml build check-code-help-to-heat && docker compose -f tests/docker-compose.yml run --no-deps --rm check-code-help-to-heat
 
+.PHONY: check-python-code-ci
+check-python-code-ci:
+	docker compose -f tests/docker-compose.yml build check-code-help-to-heat && docker compose -f tests/docker-compose.yml run --no-deps --rm check-code-ci-help-to-heat
+
```

Comment:
Think we can do some renaming across the board here: e.g. `/check-code.sh --check` reads a bit weirdly, and without the option we're fixing/formatting rather than checking (this was a problem before). So would favour something like changing all the old `check`s to `format` and then just `check-code` where you've got `check-code-ci` now.

Reply:
have done some reshuffling, think this makes a bit more sense. have updated the readme as the task to run locally has changed